### PR TITLE
Rename countries.json PD-1578

### PR DESCRIPTION
**Pull request by fezick**
Rename countries.json to be versioned and change references in the python. PD-1578.

This gets around the issue with production using the same file as QC.

### CODE REVIEW

File: seo/tests/test_import_jobs.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def test_remove_expired_jobs(self):
 
 
 class LoadETLTestCase(DirectSEOBase):
-    fixtures = ['countries.json']
+    fixtures = ['countries_167-24.json']
```
Comment (tdphillips) [inline seo/tests/test_import_jobs.py]: This is a separate file located in seo/fixtures.

Comment (fezick) [inline seo/tests/test_import_jobs.py]: Corrected.

**Comment by tdphillips** [comment]
As the code is looking for this file on s3, tests can't pass. Ideally we would stop doing remote requests for local assets but uploading the new file to s3 would also allow tests to pass.

**Comment by fezick** [comment]
I just uploaded the file.

**Pull request merged**